DigitalLearningSolutions.Web/Scripts/spec/learningMenu/fullscreen.spec.ts

```diff
@@ -105,18 +105,16 @@ describe('enterFullscreen', () => {
     expect(breadcrumbs!.classList).toContain('hidden');
   });
 
-  it('should make iframe fullscreen', () => {
+  it('should make wrapper fullscreen', () => {
     // Given
     global.document = new JSDOM(`
       <html>
       <head></head>
       <body>
-        <iframe
-          src="https://www.dls.nhs.uk/CMS/CMSContent/Course324/Section1058/Tutorials/01-01-CreateaPresentation/itspplayer.html?CentreID=101&amp;CustomisationID=19068&amp;TutorialID=4671&amp;CandidateID=254480&amp;Version=2&amp;ProgressID=285049&amp;type=learn&amp;TrackURL=https://localhost:44367/tracking/tracker"
-          class="nhsuk-grid-column-full nhsuk-u-margin-bottom-6 js-only-block"
-          height="800"
-          id="content-viewer_iframe">
-        </iframe>
+        <div class="nhsuk-u-margin-bottom-6 content-viewer_wrapper" id="content-viewer_wrapper">
+          <iframe src="https://www.dls.nhs.uk/CMS/CMSContent/Course324/Section1058/Tutorials/01-01-CreateaPresentation/itspplayer.html?CentreID=101&amp;CustomisationID=19068&amp;TutorialID=4671&amp;CandidateID=254480&amp;Version=2&amp;ProgressID=285049&amp;type=learn&amp;TrackURL=https://localhost:44367/tracking/tracker" class="nhsuk-u-margin-bottom-6 js-only-block content-viewer_iframe" id="content-viewer_iframe">

```

Comment:
Minor but I'd suggest splitting this over several lines like it was before, to make it easier to read 🙂